packages/esm-patient-registration-app/src/patient-registration/field/obs/obs-field.component.tsx

```diff
@@ -159,14 +161,43 @@ interface DateObsFieldProps {
   concept: ConceptResponse;
   label: string;
   required?: boolean;
-  dateFormat?: string;
-  placeholder?: string;
+  minDate?: string;
+  maxDate?: string;
 }
 
-function DateObsField({ concept, label, required, placeholder }: DateObsFieldProps) {
+const evaluateConfigDate = (dateString: string, dateType: string): Date | null => {
+  if (!dateString) {
+    return null;
+  }
+  if (dateString === 'today') {
+    return dayjs(new Date()).toDate();
+  }
+
+  const parsedDate = dayjs(dateString);
+  if (!parsedDate.isValid()) {
+    throw new Error(
+      translateFrom(moduleName, 'invalidObsDateErrorMessage', `The ${dateType} date value provided is invalid!`),
```

Comment:
1. Why are we using `translateFrom` here instead of `t`?
2. We can't interpolate `Min` and `Max` in here like this. It's not translatable. We can do something like `t('invalidObsDateMinValue', 'The minimum date value provided is invalid.')`

Reply:
It would, of course, be better to include some example of what is valid.